Rampastring.Tools.csproj

```diff
@@ -1,74 +1,40 @@
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <DocumentationFile>bin\Release\Rampastring.Tools.xml</DocumentationFile>
+    <RootNamespace>Rampastring.Tools</RootNamespace>
+    <ComVisible>false</ComVisible>
+    <CLSCompliant>false</CLSCompliant>
+    <AnalysisLevel>preview-all</AnalysisLevel>
+    <LangVersion>preview</LangVersion>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <Platforms>AnyCPU</Platforms>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+    <ComVisible>false</ComVisible>
+    <CLSCompliant>false</CLSCompliant>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Windows.Forms" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
+    <None Include="README.md" Pack="true" PackagePath="\" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="BooleanStringStyle.cs" />
-    <Compile Include="Conversions.cs" />
-    <Compile Include="ExtendedStringBuilder.cs" />
-    <Compile Include="IIniFile.cs" />
-    <Compile Include="IIniSection.cs" />
-    <Compile Include="IniFile.cs" />
-    <Compile Include="IniParseException.cs" />
-    <Compile Include="INIProperties\BoolProperty.cs" />
-    <Compile Include="INIProperties\DoubleProperty.cs" />
-    <Compile Include="INIProperties\EnumProperty.cs" />
-    <Compile Include="INIProperties\GenericINIProperty.cs" />
-    <Compile Include="INIProperties\IIniProperty.cs" />
-    <Compile Include="INIProperties\IntProperty.cs" />
-    <Compile Include="INIProperties\StringProperty.cs" />
-    <Compile Include="IniSection.cs" />
-    <Compile Include="Logger.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Utilities.cs" />
-    <Compile Include="WindowFlasher.cs" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.435">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
I'd prefer it without StyleCop as I have my own established code style partially using the VS document formatter and mostly find such analyzers an annoyance.